events/lastAvatarStandingTrivia/entityScripts/triviaColorCheckBlue.js

```diff
@@ -0,0 +1,114 @@
+// triviaColorCheckBlue.js
+
+//  Created by Mark Brosche on 11/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+/* global EventBridge Users AccountServices */
+
+(function(){
+    var _this,
+        bubble,
+        gameOn,
+        gameZone;
+    
+    var DISQUALIFIED_POSITION = { x: 84.8138, y: 12.5, z: 88.744 },
```

Comment:
These constants should be inside some imported JSON file, no? They're repeated across a bunch of scripts

Reply:
That would only make sense!  I'll fix that.